sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -85,6 +87,13 @@ case class CreateDataSourceTableCommand(table: CatalogTable, ignoreIfExists: Boo
       }
     }
 
+    table.provider.get.toLowerCase match {
```

Comment:
We are able to check here for a normal `CREATE TABLE`.

Reply:
This just covers CREATE DATA SOURCE TABLES. What happened for CREATE HIVE SERDE TABLES?

Reply:
Ya. That's a good point!

Reply:
@gatorsmile . I have a question. Do we have an issue on Hive SERDE table?
```
CREATE TABLE t(`a b` INT) USING hive OPTIONS (fileFormat 'parquet')
```
I thought Hive schema is preferred than Parquet schema.

Reply:
What is Hive schema? What is Parquet schema? 

Could you insert rows to the table you created?

Reply:
Oh, I see. It fails.
```scala
scala> sql("set spark.sql.hive.convertMetastoreParquet=false")
res5: org.apache.spark.sql.DataFrame = [key: string, value: string]

scala> sql("INSERT INTO t VALUES(1)")
17/09/05 11:34:03 ERROR Utils: Aborting task
org.apache.hadoop.hive.ql.metadata.HiveException: java.lang.IllegalArgumentException: field ended by ';': expected ';' but got 'b' at line 1:   optional int32 a b
	at org.apache.hadoop.hive.ql.io.HiveFileFormatUtils.getHiveRecordWriter(HiveFileFormatUtils.java:249)
	at org.apache.spark.sql.hive.execution.HiveOutputWriter.<init>(HiveFileFormat.scala:123)
```